nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EvaluateXPath.java

```diff
@@ -125,6 +132,14 @@
             .defaultValue(RETURN_TYPE_AUTO)
             .build();
 
+    public static final PropertyDescriptor VALIDATE_DTD = new PropertyDescriptor.Builder()
+            .name("Validate DTD")
+            .description("Specifies whether or not the XML content should be validated against the DTD.")
+            .required(false)
```

Comment:
A select type property with a default value is usually defined as 'required(true)'. If not, 'No value' can be selected from the pull down menu, but it will be overwritten by the default value.